TappNetwork/facts-and-figures-web
# Facts & Figures 2022 Updates

# Details

fixed mappings in src/data/mappings.json to conform to 2022 data in excel spreadsheet

# Steps to test changes

1. Clone Repo.
2. run 'npm run start' in terminal
3. confirm in browser that 2022 facts and figures excel has loaded correctly into app. 

# Issues Addressed

[Facts & Figures 2022 Updates](https://app.clickup.com/t/2je18qp)

# Screenshots

Any visual changes must have screenshots
![Screenshot from 2022-03-25 13-24-02](https://user-images.githubusercontent.com/29612767/160188039-e038bd89-b82a-4f21-b748-2b6fefe24569.png)
